package.json

```diff
@@ -157,14 +157,14 @@
     "react-grid-layout": "^0.16.2",
     "react-input-range": "^1.3.0",
     "react-markdown": "^3.1.4",
-    "react-redux": "^5.0.6",
+    "react-redux": "^5.0.7",
     "react-router-dom": "4.2.2",
     "react-sizeme": "^2.3.6",
     "react-toggle": "4.0.2",
     "reactcss": "1.2.3",
-    "redux": "3.7.2",
+    "redux": "4.0.0",
```

Comment:
__optional nit:__ we try to use "caret" versions for all dependencies we add and all dependencies we update since we started to use `yarn.lock`. But I see that it's used without caret in x-pack, so feel free to keep it as is, or update to "caret" versions in both `package.json` files.